agent/config.go

```diff
@@ -625,6 +625,10 @@ type Config struct {
 	// true, we ignore the leave, and rejoin the cluster on start.
 	RejoinAfterLeave bool `mapstructure:"rejoin_after_leave"`
 
+	// CheckEnableExec controls whether health checks which execute scripts
+	// are enabled. This includes regular script checks and Docker checks.
+	CheckEnableExec bool `mapstructure:"check_enable_exec"`
```

Comment:
That's good feedback. I went around on this a bit and sort of went with something that would go with a check in structure if we ever made one but your suggestion fits more with what we have. I'll change this.